app/api/groups/get_group.go

```diff
@@ -98,26 +98,34 @@ func (srv *Service) getGroup(w http.ResponseWriter, r *http.Request) service.API
 
 	query := srv.Store.Groups().
 		Joins(`
-			LEFT JOIN groups_ancestors_active
-				ON groups_ancestors_active.child_group_id = groups.id AND
-					groups_ancestors_active.ancestor_group_id = ?`, user.OwnedGroupID).
+			LEFT JOIN (
+				SELECT 1 AS found
+				FROM group_managers
+				JOIN groups_ancestors_active
+					ON groups_ancestors_active.ancestor_group_id = group_managers.group_id AND
+						groups_ancestors_active.child_group_id = ?
+				WHERE group_managers.manager_id = ?
+				LIMIT 1
+			) AS manager_access ON 1`, groupID, user.GroupID).
```

Comment:
Actually not. The suggested JOIN can produce more than one row (e.g. if the user is a manager for the group itself and for one of its ancestors) while the original one cannot produce more than one row because of the 'LIMIT 1'.

Reply:
```

Reply:
But as we have 'LIMIT 1' on the top level anyway and the suggested JOIN seems to work better (see above), we'll use it.

Reply:
Changed the code